src/controllers/todo.controller.ts

```diff
@@ -17,9 +17,16 @@ const getAllData = async(req:Request,res:Response)=>{
     res.status(200).json({data:allData});
 }
 
+const single = async(req:Request,res:Response)=>{
+    const {id} = req.params;
+    const intId = parseInt(id);
+    const blog = await getSingletask(intId);
+    res.status(200).json({blog:blog});
+}
```

Comment:
- [ ]  You've done a great job, but there is something I recommend you to change. Try to use the right names that concern your project. so that it won't seemed like you've used other's codes.